Add "Markup Shorthands: css no"

wantehchang opened the pull request:
Turn off the dfn autolinks for CSS types so that incorrect links won't be created for fourCC's such as 'clap' and 'colr'.


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/wantehchang/av1-isobmff/pull/172.html" title="Last updated on May 15, 2023, 9:54 PM UTC (6d81147)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-isobmff/172/e95f6e0...wantehchang:6d81147.html" title="Last updated on May 15, 2023, 9:54 PM UTC (6d81147)">Diff</a>

cconcolato commented:
Looking at the diff https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-isobmff/172/e95f6e0...wantehchang:6d81147.html#index-defined-elsewhere I see that the section with "terms defined in other specifications" is now missing the code points for mdcv, colr, ... Is there a way to fix that?

wantehchang commented:
I don't know. I am not familiar with these markup shorthands.

Another solution is to remove the single quotes from the "Changes since v1.2.0 release" section. Or perhaps replace the single quotes with backticks or double quotes.

cconcolato commented:
I think it's because of this code https://github.com/AOMediaCodec/av1-isobmff/pull/172/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R20-R30 It says they are 'property'. If we were to use 'dfn' or else, that should work again after your change.

wantehchang commented:
Cyril: I tried what you suggested in the meeting today, but I still could not get it to work properly. So I decided to abandon this pull request and switch to a simple solution: just remove the single quotes from the changelist section (https://github.com/AOMediaCodec/av1-isobmff/pull/173).
